.github/workflows/run-tests.yml

```diff
@@ -325,6 +331,54 @@ jobs:
           --results-directory ${{ github.workspace }}/testresults
           --timeout ${{ inputs.testSessionTimeout }}
           ${{ inputs.extraTestArgs }}
+          ${{ inputs.ignoreTestFailures && '|| true' || '' }}
+
+      - name: Verify test results exist
```

Comment:
@radical  can you help me understand why we need this? What are we protecting against? Is it to know when the test run crashed as opposed to just it failed some tests?

Reply:
This is because we are ignoring the exit code from the test run itself in the previous step. The runner returns non-zero exit code when tests fail, and it would be non-zero if anything else fails like arguments are incorrect.
So, we rely on looking for the trx files to confirm that the runner did run and completed writing its results.